advanced_communication_options/update_comms_settings.py

```diff
@@ -125,8 +196,8 @@ def check_pi3():
 	if hw_id=="BCM2709":
```

Comment:
@CleoQc  Actually, if you take a look here, you can see that `BCM2709` is actually the chip that `RPI 2` has.
Take a look in `Adafruit's` version and you'll see the difference.